feat/gene/gene.go

```diff
@@ -220,39 +220,43 @@ func (t *CodingTranscript) Orientation() feat.Orientation {
 
 // UTR5start returns the start of the 5'UTR relative to the transcript.
 func (t *CodingTranscript) UTR5start() int {
-	if t.Orient == feat.Forward {
+	ori, _ := feat.BaseOrientationOf(t)
+	if ori == feat.Forward {
 		return 0
-	} else if t.Orient == feat.Reverse {
+	} else if ori == feat.Reverse {
 		return t.CDSend
```

Comment:
>Nope they should not be mapped through `BasePositionOf`.

Ahh, yes, the docs say relative to the transcript. I don't like that, but it is already there.

Why don't I like it? Say you have the following as described in your original issue:

```
Transcript:          3'<------------5'       Forward
Gene:              <---------------------    Reverse
Chrom:       -----------------------------------
```

and you want the 5'UTR as mapped to the chromosome; we often do RTE/lnc analysis based on this kind of criterion. It's also at odds with the rest of the position API.